### DISCUSSION TITLE
Way to auto focus in a component

### DISCUSSION THREAD

**Discussion by JamesMolina007** [Foblex/f-flow#70] [General]
Hi, Im creating a flow with foblex. I would like to know is there is a easy way to focus in a specific node
![image](https://github.com/user-attachments/assets/dc758818-1d03-47f8-a03c-c3196546e808)

For example, I could have a lot of flows, and when I click in one of them, I would like to focus in the main component

**Comment by siarheihuzarevich** [comment]
@JamesMolina007  do you mean that you want to center a specific node?

**Reply by siarheihuzarevich** [reply]
If yes, please use centerNodeOrGroup api https://flow.foblex.com/docs/f-canvas-component

**Reply by JamesMolina007** [reply]
Okay awesome
